src/components/MaxHeapVisualizer.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div
+    id="max-heap-visualizer"
+    class="flex tree-container"
+  >
+    <v-btn
+      class="back md-raised md-primary"
+      @click="goBack"
+    >
+      Go back
+    </v-btn>
+    <div class="title">
+      <h1 class="name">
+        Structure: Max Heap
+      </h1>
+    </div>
+    <div class="flex buttons">
+      <InsertInput
+        :disabled="isDisabled"
+        @myEvent="insertInputEvent"
+      />
+      <DeleteInput
+        :onlypop="onlypop"
```

Comment:
Maybe we can change the text in the DeleteInput component to Pop, when onlypop = true

Reply:
I don't really understand what this flag is for

Reply:
Heap don't use classical delete(key) function. They only pop the node with the higher (or lower) priority. So te hide the text input in the DeleteInputComponent, we use that flag

Reply:
Ah ok, I think the current UI is sophisticated and lets the user know that they don't have choice of element when they are removing from the heap. But if you feel like it, go ahead.